test/models/vote_test.rb

```diff
@@ -0,0 +1,22 @@
+require "test_helper"
+
+describe Vote do
+  let(:vote) {votes(:one)}
+
+  it "will be valid when user and work are present" do
```

Comment:
You should have a test to make sure that the combination of user/work ids are unique.